_pages/en/strategy-learning-automating-improving.md

```diff
@@ -54,68 +55,20 @@ IITB teams should:
 ##### Learn
 
 - Stay up to date on trends and technologies inside and outside the GC
-- Teach yourself basic programming. Examples of instructional material can be found [here](#Learn).
-- Experiment with cloud technologies. Click [here](#Learn) to learn how to create a cloud sandbox environment.
-  - Ex: [Cousera](https://www.coursera.org/), [Udemy](https://www.udemy.com/), or [MOOCs](https://www.mooc-list.com/)
-- Read books, articles or documentation. Examples of content can be found [here](https://sara-sabr.github.io/ITStrategy/references-en.html).
+- Teach yourself basic programming
```

Comment:
Should make it easy for anyone, add examples with free online courses:
(e.g. .Net Core, Java, Python, JavaScript)

Reply:
Added